src/components/header/Header.js

```diff
@@ -0,0 +1,38 @@
+import { render } from '../../shared/dom';
+
+const template= (close, open) =>`
```

Comment:
You are not using this close and open functions,

Convert this back to be an const instead od function, just plain html :p